### DISCUSSION TITLE
Default status for new issues

### DISCUSSION THREAD

**Discussion by jdshkolnik** [community/community#4932] [Projects and Issues] [answered]
Is there a plan for defining the default status for newly created issues? That plus it should be definable as a required value.

**Comment by erikr** [comment]
Agree on need for this functionality. I would like new issues to be automatically assigned to the backlog column.

**Comment by douglaslassance** [comment]
This feature should be available for any single select field I would say.

**Comment by azenMatt** [comment, marked as answer]
Hi everyone - this is now available via the 'workflow' tab - check it out in the top right of your projects 😄

**Reply by mrmurphy** [reply]
Correct me if I'm wrong, but I think this only applies to issues and PRs, not cards. I'd love to be able to specify a default status for my cards as well.

**Reply by azenMatt** [reply]
Do you mind expanding a bit more on what you mean by a card here? 

If this is adding an item to the board layout - as you can directly pick which status you add to (by picking the column) we don't apply the automation on the assumption that because you have added it directly you want it to go there specifically.

**Reply by mrmurphy** [reply]
That makes sense. Sorry, yes. With "cards" I was referring to "item"s. In my case, I was adding items in the list layout, because I wanted to add items to a specific group. Looks like I currently can only have one field with a default status, either the field I'm grouping by in list layout, or the status field in the board layout. Is that correct?

**Comment by jdshkolnik** [comment]
@azenMatt I enabled the "Item added to project" workflow but converting a new item into an issue didn't seem to trigger it.

**Reply by azenMatt** [reply]
Thanks @jdshkolnik (Hey John - I didn't realise that was you until I saw your name in the email notification! I hope all is going well with you) this is a bug, we will get it fixed. 

Bug 1 - draft issues, when added to a project aren't triggering the 'item added to project' workflow 
Bug 2 - convert to issue doesn't trigger the 'item added to project' workflow

**Comment by donut** [comment]
I'd like the option to have draft issues automatically assigned a Status when created.
